package.json

```diff
@@ -78,6 +80,7 @@
     "*.{js,ts,jsx,tsx,css,md}": "prettier --write"
   },
   "dependencies": {
-    "lodash": "^4.17.21"
+    "lodash": "^4.17.21",
+    "react-router-dom": "^5.3.0"
```

Comment:
:thinking: Should this go in `devDependencies` (for the example) and `peerDependencies` (when it's used downstream) like `React`?

Since we're actually rendering components from the library, it maybe should belong in `dependencies`. :thinking: I guess I'm not sure.

Reply:
I'm not sure either :sweat_smile: